junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ParallelExecutionConfiguration.java

```diff
@@ -60,4 +61,17 @@
 	 */
 	int getKeepAliveSeconds();
 
+	/**
+	 * Get the predicate called when the {@code ForkJoinPool} is saturated.
+	 * This occurs when the ForkJoinPool has reached MaxPoolSize
+	 * and there are no worker threads available. The predicate
+	 * should return true if the calling thread should block.
+	 * If the predicate is null or returns false a RejectedExcecutionException
+	 * is thrown.
+	 * @return a Predicate accepting a ForkJoinPool as the parameter or null.
+	 * @since 1.9
```

Comment:
```suggestion
	 * @since 1.9
	 * @see ForkJoinPool#ForkJoinPool(int, ForkJoinPool.ForkJoinWorkerThreadFactory, Thread.UncaughtExceptionHandler, boolean, int, int, int, Predicate, long, TimeUnit)
```